mmdet/models/losses/iou_loss.py

```diff
@@ -466,3 +519,61 @@ def forward(self,
             avg_factor=avg_factor,
             **kwargs)
         return loss
+
+
+@MODELS.register_module()
+class EIoULoss(nn.Module):
+    r"""Implementation of paper `Extended-IoU Loss: A Systematic
+    IoU-Related Method: Beyond Simplified Regression for Better
+    Localization <https://ieeexplore.ieee.org/abstract/document/9429909>`_
+
+    Code is modified from https://github.com//ShiqiYu/libfacedetection.train.
+
+    Args:
+        pred (Tensor): Predicted bboxes of format (x1, y1, x2, y2),
+            shape (n, 4).
+        target (Tensor): Corresponding gt bboxes, shape (n, 4).
+        smooth_point (float): hyperparameter, default is 0.1.
+        eps (float): Eps to avoid log(0).
+    Return:
+        Tensor: Loss tensor.
```

Comment:
The order of the args should be consistent with the __init__ method.

And delete the "Return"